Solutions/ISC Bind/Parsers/ISCBind.txt

```diff
@@ -0,0 +1,74 @@
+// Title:           ISC Bind
+// Author:          Microsoft
+// Version:         1.0
+// Last Updated:    09/07/2022
+// Comment:         Inital Release
+//  
+// DESCRIPTION:
+// This parser takes raw ISC Bind logs from a Syslog stream and parses the logs into a normalized schema.
+//
+// USAGE:
+// 1. Open Log Analytics/Azure Sentinel Logs blade. Copy the query below and paste into the Logs query window. 
+// 2. Click the Save button above the query. A pane will appear on the right, select "as Function" from the drop down. Enter a Function Name.
+//    It is recommended to name the Function Alias, as ISCBind
+// 4. Kusto Functions can typically take up to 15 minutes to activate. You can then use Function Alias for other queries.
+//
+// REFERENCES: 
+// Using functions in Azure monitor log queries: https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// 
+// LOG SAMPLES:
+// This parser assumes the raw log are formatted as follows:
+//
+// <190>Jun 30 16:15:59 218.254.206.240 named[22442]: client @0a0a00000a0a00 247.69.221.212#35306 (abc-exam.d.example.com): view 7: query: test123.example.com IN A -E(0)DCK (218.254.206.240)
+// <190>Jun 30 16:15:59 218.254.206.240 named[22442]: 30-Jun-2022 16:15:59.366 client @0a0a00000a0a00 247.69.221.212#35306 (abc-exam.d.example.com): view 7: UDP: query: test.example.com IN A response: NXDOMAIN -AE
+
+let request = Syslog 
```

Comment:
Parser is not working for all the inputs.

Reply:
Updated the parser.